internal/approval_test.go

```diff
@@ -134,6 +134,77 @@ func TestContentsUrlToRelDir(t *testing.T) {
 	})
 }
 
+func TestSplitMembers(t *testing.T) {
+	tests := map[string]struct {
+		commits   []*github.RepositoryCommit
+		members   []*github.User
+		allowed   []string
+		dismissed []string
+	}{
+		"When no member is an author in PR": {
+			[]*github.RepositoryCommit{
+				{
+					Committer: &github.User{Login: github.String("foo")},
+				},
+			},
+			[]*github.User{
+				{Login: github.String("bar")},
+			},
+			[]string{"bar"},
+			nil,
+		},
+		"When only member is an author in PR": {
+			[]*github.RepositoryCommit{
+				{
+					Committer: &github.User{Login: github.String("foo")},
+				},
+			},
+			[]*github.User{
+				{Login: github.String("foo")},
+			},
+			nil,
+			[]string{"foo"},
+		},
+		"When multiple members exist without being author": {
+			[]*github.RepositoryCommit{},
+			[]*github.User{
+				{Login: github.String("foo")},
+				{Login: github.String("bar")},
+				{Login: github.String("baz")},
+			},
+			[]string{"foo", "bar", "baz"},
+			nil,
+		},
+		"When multiple members exist, some are authors": {
```

Comment:
Just double-checking: this would cover the case then a single commit is co-authored by two (or more) people?